### PR TITLE
Handle errors from AskPlaces API and track AskPlaces usage in Amazon Pinpoint

### PR DESCRIPTION
**Pull request by marinrelatic**
*Description of changes:*
- Handle errors from AskPlaces API
- Track AskPlaces usage in Amazon Pinpoint
- Added `us-west-2` region to Content-Security-Policy for Pinpoint and PublicAppService APIs

Screenshot 2023-11-21 at 4 49 32 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
 		/>
```
Comment (TheRBajaj) [inline index.html:27]: if `https://*.execute-api.us-west-2.amazonaws.com` is for testing purposes, then I suggest we remove it as we should only include url's here that make it to prod.

File: package-lock.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 				"@aws-amplify/ui-react": "3.6.1",
 				"@aws-sdk/credential-providers": "3.370.0",
 				"@turf/turf": "6.5.0",
-				"@types/react-window": "^1.8.5",
+				"@types/react-window": "1.8.5",
```
Comment (TheRBajaj) [inline package-lock.json:14]: are the modifications on package.json fine or should we not modify this file unless we want to bring in a new dependency?

File: index.html (hunk 1 above)
Comment (marinrelatic) [reply index.html:27]: API can be hosted in multiple regions

File: package-lock.json (hunk 2 above)
Comment (marinrelatic) [reply package-lock.json:14]: This is `package-lock.json`, not `package.json`

File: index.html (hunk 1 above)
Comment (marinrelatic) [reply index.html:27]: It is required in pre prod stages and there is no security risk in adding all regions supported by Amazon Location Service

**Review by thehoneymad — APPROVED** [review]

**Comment by thehoneymad** [comment]
🚀

**Pull request merged**